lang/yabasic/Portfile

```diff
@@ -15,11 +15,18 @@ long_description	Yabasic implements the most common and simple elements \
 
 homepage		http://www.yabasic.de/
 master_sites	${homepage}/download/
-checksums		md5 98f0cb59db973b89753abcc24b2c5ec2
+checksums		md5 2632ddb3e39fc462c7b75999324a80e0 \
+				rmd160 bd61bda74f0fe7ddf6039284d077dcde891a52cb \
+				sha256 17aa9bd50ce98ac5d4f66edae54ff1d13405578757cccd6d861a37c4165b6773 \
+				size 707361
```

Comment:
Please fix the formatting to align everything. Probably tabs need to be converted to spaces.

Reply:
Reformatted the Portfile with spaces.